runtimes/tensorflow/ubi9-python-3.12/Dockerfile.cuda

```diff
@@ -71,6 +71,9 @@ WORKDIR /opt/app-root/bin
 COPY ${TENSORFLOW_SOURCE_CODE}/pylock.toml ./
 # Copy Elyra dependencies for air-gapped enviroment
 COPY ${TENSORFLOW_SOURCE_CODE}/utils ./utils/
+# Download Elyra boostrapper.py
+RUN curl -fL https://raw.githubusercontent.com/opendatahub-io/elyra/refs/tags/v4.3.0/elyra/kfp/bootstrapper.py \
```

Comment:
isn't this bootstrapper installed together with the odh-elyra package? Can't we grab it from python's site-packages? And if it is not installed, can't we install it that way?